src/se05x/commands.toml

```diff
@@ -89,7 +89,7 @@ p2 = "P2_DEFAULT"
 
 [scp_external_authenticate.payload]
 then = [
-  { name = "host_cryptogram", type = "[u8; 8]" }
+  { name = "host_cryptogram", type = "[u8; 8]" },
```

Comment:
Wait does that mean that the previous toml libraries accepted invalid input?

Reply:
At least the generated file did not change.  I haven’t looked into the details yet.

Reply:
I don’t see a good reason why `toml` would have accepted that.  But it also looks like the package is not maintained anymore.  So if we ever need toml support before Python 3.11 in the future, we should prefer `tomli` over `toml`.